packages/api/src/ApiContext.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```

Comment:
Noticed some `import`s are single quote and others double--personally I prefer single, but I don't mind as long as we're consistent I think.